dockerfiles/Dockerfile.cuda

```diff
@@ -5,16 +5,20 @@
 # Dockerfile to run ONNXRuntime with CUDA, CUDNN integration
 
 # nVidia cuda 11.4 Base Image
-FROM nvcr.io/nvidia/cuda:11.4.1-cudnn8-devel-ubuntu18.04
+FROM nvcr.io/nvidia/cuda:11.4.2-cudnn8-devel-ubuntu20.04
+#FROM nvidia/cuda:11.4.2-devel-ubuntu20.04
```

Comment:
remove comment?

Reply:
Hi. Apologies! ill clean up the comments - I missed these notifications. Thank you.